### PR TITLE
mail-filter/bogofilter: removed wrong license

### PR DESCRIPTION
**Pull request by scardracs**
Package-Manager: Portage-3.0.20, Repoman-3.0.3
Signed-off-by: Marco Scardovi <marco@scardovi.com>

### CODE REVIEW

File: mail-filter/bogofilter/bogofilter-1.2.5-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+EAPI=7
+
+inherit autotools db-use flag-o-matic toolchain-funcs
+
+DESCRIPTION="Bayesian spam filter designed with fast algorithms, and tuned for speed"
+HOMEPAGE="http://bogofilter.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.xz"
+
+LICENSE="GPL-3+ CC-BY-ND-3.0"
```
Comment (thesamesam) [inline mail-filter/bogofilter/bogofilter-1.2.5-r1.ebuild]: This still has the wrong licence? Anyway, I’d just git mv the bad ebuild and fix it in one commit?

**Review by thesamesam — CHANGES_REQUESTED** [review]

**Review by juippis — COMMENTED** [review]
I don't think a revbump is required for this change...

**Comment by scardracs** [comment]
> I don't think a revbump is required for this change...

Done

**Review by juippis — APPROVED** [review]
Thanks!

**Pull request closed without merging**